hypothesis/lms
# Add Microsoft OneDrive backend settings

Fixes https://github.com/hypothesis/lms/issues/2941

This includes the config for Microsoft OneDrive when doing assignment configuration etc.

### Testing notes

 * Run `make devdata`
 * Go to: https://hypothesis.instructure.com/courses/125/assignments/1257
 * View source on our frame and search for "microsoftOneDrive"
 * You should find: `"microsoftOneDrive": {"enabled": false}`
 * Goto: http://localhost:8001/admin/
 * Search for: `Hypothesisb3be0b33d0b5e4b1cf3aaf04e0e1819a`
 * Check "Microsoft OneDrive enabled"
 * Refresh the view source page
 * You should find `"microsoftOneDrive": {"clientId": "<CLIENT_ID_HERE>", "enabled": true}`
